### ISSUE TITLE
Support Yandex.Browser

### ISSUE THREAD

**Issue by ingria** [RealFaviconGenerator/realfavicongenerator#94]
Hello! I suggest supporting [Yandex.Browser](http://browser.yandex.com/?lang=en)'s tableau. YaBrowser is rather popular in Russia and other CIS countries (7-8% of users), and has one cool feature: tableau. This is something like speed dial, but more convinient and flexible. 

![tableau](https://cloud.githubusercontent.com/assets/2178368/4651451/fbf8caf6-549b-11e4-8ae8-532b58ed9f42.png)

**Documentation**: http://api.yandex.com/tableau/doc/

**TL;DR:** 
it needs a meta tag:

``` html
<link rel="yandex-tableau-widget" href="manifest.json" />
```

And `manifest.json` file with a logo:

``` json
{
    "version": "1.0",
    "api_version": 1,

    "layout": {
        "logo": "/logo.png",
        "color": "#FFFFFF",
        "show_title": "none"
    }
}
```

**Comment by phbernard** [comment]
I didn't knew that! Thank you for the suggestion!

**Comment by phbernard** [comment]
Two steps:
- Add Yandex Browser support in the API. Done and deployed a minute ago.
- Add Yandex support in the UI. Todo later.
